tests/testYearWrap.py

```diff
@@ -26,13 +26,19 @@
 import unittest
 import lsst.utils.tests
 import lsst.ctrl.stats.records as recordslib
-from datetime import date
+import datetime
+from dateutil import tz, parser
 from lsst.ctrl.stats.reader import Reader
 
 
 def setup_module(module):
     lsst.utils.tests.init()
 
+def tolocal(dateVal):
```

Comment:
Dumb question:  Would you suggest that I move that code to the regular package, or put the __init__.py in tests?  I resisted putting __init__.py in there in the first place because it didn't seem right for a tests directory, and I'm leaning towards moving it to the regular package and importing from there.